src/main/java/racingcar/controller/Game.java

```diff
@@ -0,0 +1,56 @@
+package racingcar.controller;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.ArrayList;
+import java.util.List;
+import racingcar.model.Car;
+import racingcar.model.Player;
+import racingcar.model.WinnerCar;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+public class Game {
+    private static final int FIRST_BOUND = 0;
+    private static final int LAST_BOUND = 9;
+    private static final int MOVE_LEAST_BOUND = 4;
+    private final Player player = new Player();
+    private final InputView inputView = new InputView();
+    private final OutputView outputView = new OutputView();
+    private final List<Car> cars;
+
+    public Game() {
+        cars = new ArrayList<>();
+        player.setCarName(inputView.askCarName());
+        player.setAttemptNumber(inputView.askAttemptNumber());
```

Comment:
setter는 최대한 지양해야 한다고 생각해요!!
예를 들어 지금처럼 setter를 사용했을 때 단점은 
  1. Player 객체에 `carName`과 `attemptNumber`라는 멤버변수가 모두 바깥으로 노출되고,
  2. 만약 setter를 통해 값을 할당해주지 않은 채로 누군가가 getter로 꺼낸다면 NPE 위험이 있다는 점이 있을 것 같아요.

setter를 사용하지 않고 player 생성자에서 필요한 자원을 전달받도록 하면 좋을 것 같아요!
```suggestion
        new Player(inputView.askCarName(), inputView.askAttemptNumber());
```